### ISSUE TITLE
Add benchmark to gemspec to silence Ruby 3.5 warning

### ISSUE THREAD

**Issue by jaryl** [aderyabin/sniffer#74]
Hello,

I'm seeing the following warning when using `sniffer` (version 0.5.0) on Ruby 3.4.0:

```
/usr/local/bundle/ruby/3.4.0/gems/sniffer-0.5.0/lib/sniffer/adapters/net_http_adapter.rb:4: warning: /usr/local/lib/ruby/3.4.0/benchmark.rb was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.5.0.
You can add benchmark to your Gemfile or gemspec to silence this warning.
```

It seems `sniffer` uses `benchmark` which will be removed from default gems in Ruby 3.5.0. It would be great to add `benchmark` as a runtime dependency in the gemspec to future-proof the gem and silence this warning.

Thanks!
